enterprise/frontend/src/embedding-sdk/components/public/StaticDashboard/StaticDashboard.tsx

```diff
@@ -55,27 +57,29 @@ const _StaticDashboard = ({
     });
 
   const { hasNightModeToggle, isNightMode, onNightModeChange, theme } =
-    useEmbedTheme(options.theme);
+    useEmbedTheme();
```

Comment:
Do we show a button to toggle the theme in the embedded static dashboard? Since I see `onNightModeChange` here, but not sure if that is a no-op in this case. (Since we want SDK users to rely on the custom theming, rather than the display theme)

Reply:
If you go fullscreen with the SDK dashboard, there's a toggle for night mode. It's the same behavior as that of public or embedded dashboards, but we can disable the toggle with a flag within `useEmbedTheme`

Reply:
What do you think? Should we just remove the option for now?